### ISSUE TITLE
Blackheart the Inciter - Shadow Labirynth

### ISSUE THREAD

**Issue by kikomir** [ccshiro/cc-buglist#693]
As far as I know his spell "Incite Chaos" should affect all party members and is not resistible. On 1-2 occasions it did not affect me at all. If this is intended behavior please ignore this report. Also the players' damage during this mind control phase seemed way too high but I couldn't research this more thoroughly because the combat log wasn't working.

edit: forgot to mention, players also say "cancel auto" out loud during the mind control phase.

**Comment by ccshiro** [comment]
I couldn't actually reproduce it, but reasoning what can go wrong I made some changes. This needs testing to see if I got the bug or not.

**Comment by KimmoKM** [comment]
Yes, the damage during MC phase seems too high. Or rather, the casting AI seems too good. I wiped on the boss a dozen times just now and my character (a fire mage) consistently zeroed in on someone and just cast Fire Ball after Fire Ball. If I was out of sight of other players (like I tried to be for obvious reasons), the character ran straight into range while spamming Arcane Explosion/Blast Wave and then started nuking again. We didn't defeat the boss until adjusting our party composition away from 3 mages and holy paladin healer (a setup that was far from ideal against Blackheart, that of course was also a factor) and even after that it took a couple of wipes. I unfortunately messed up the video recording of the attempts so I can't provide video evidence.

Now, it's perhaps a bit difficult to draw conclusions [from](https://www.youtube.com/watch?v=imIMHVXNJi0) [retail](https://www.youtube.com/watch?v=4M6g6mHBXFM) [YouTube](https://www.youtube.com/watch?v=QR3B0Uivjp4) [videos](https://www.youtube.com/watch?v=uQ8BaksIbcs) because there's a major bias: none of the videos show wipes and successful kills clearly didn't have outrageously bad RNG, but they are in-line with my memory from retail: channelled Blizzard/Hurricane/Rain of Fire/Volley, casters autoattacking rather than casting their main nuke, players running around to random direction instead of chaincasting nukes or attacking someone in melee without ever switching a target, mages casting Pyroblast and never getting the cast off, shamans refreshing Lightning shield and other such silliness. Of course, players were killed from time to time due to bad RNG, but if my memory serves me correctly, the majority of wipes in retail were a result of threat drops and phase transitions being handled poorly, not party members being consistently killed halfway into the MC.
